Makefile.toml

```diff
@@ -129,3 +129,38 @@ description = "Clean up WASM build artifacts"
 category = "ICU4X WASM"
 command = "rm"
 args = ["-rf", "wasmpkg"]
+
+[tasks.bincode-clean]
+description = "Clean out the bincode data."
+category = "ICU4X Bincode"
+script_runner = "@duckscript"
+script = '''
+# Use duckscript, rather than a unix command, so that this works on Windows.
+rm ./resources/testdata/data/bincode
+'''
+
+[tasks.bincode-gen-testdata-no-clean]
+description = "Generate bincode testdata without removing the old data"
+category = "ICU4X Bincode"
+cwd = "components/provider_fs"
+command = "cargo"
+args = [
+    "run",
+    "--features", "export-bin",
+    "--features", "bincode",
+    "--features", "serialize_none",
+    "--",
+    "--cldr-tag", "38.1.0",
+    "--out", "../../resources/testdata/data/bincode",
+    "--all-keys",
+    "-s", "bincode",
+]
+
+[tasks.bincode-gen-testdata]
```

Comment:
This hides the complexity of all of the command line arguments, and provides a single command to generate bincode. I found this helpful when testing my skeletons work.

Reply:
I think I would make it its own job. #315 will allow us to share artifacts between jobs so that GitHub doesn't need to reinstall cargo-make several times.